src/FrontDoor/FrontDoor/Cmdlets/NewAzureRmFrontDoor.cs

```diff
@@ -106,6 +114,23 @@ public override void ExecuteCmdlet()
                                 Name,
                                 ResourceGroupName));
             }
+
+            Management.FrontDoor.Models.BackendPoolsSettings backendPoolsSettings;
+            switch (ParameterSetName)
+            {
+                case FieldsWithCertificateNameCheckParameterSet:
+                {
+                    backendPoolsSettings = new Management.FrontDoor.Models.BackendPoolsSettings(
+                            DisableCertificateNameCheck ? PSEnforceCertificateNameCheck.Disabled.ToString() : PSEnforceCertificateNameCheck.Enabled.ToString());
+                    break;
+                }
+                default:
+                {
+                    backendPoolsSettings = BackendPoolsSettings?.ToSdkBackendPoolsSettings();
```

Comment:
>backendPoolsSettings [](start = 20, length = 20)

What happens if backendpoolssettings are not passed? Will RP handle null backendpoolsSettings?

Reply:
Add a test case for this.

---
In reply to: [330339054](https://github.com/jessicl-ms/azure-powershell/pull/1#discussion_r330339054) [](ancestors = 330339054)